### ISSUE TITLE
[Feature]: Add "MINT-Schwerpunkte" to organization

### ISSUE THREAD

**Issue by nac62116** [mint-vernetzt/community-platform#160]
### What do you like to be improved?

- [x] #187
- [x] #188

**Comment by phollome** [comment]
```
[
  'Mädchen und Frauen in MINT',
  'Netzwerkarbeit',
  'Social Media Management',
  'Gute Praxis',
  'MINT-Forschung',
  'Innovation',
  'Politik',
  'Social Media Managament',
  'Berufsorientierung',
  'frühkindliche Bildung',
  'schulische MINT-Bildung',
  'Ländlicher Raum' 
]
```

**Comment by nac62116** [comment]
Implemented in #229 , #187 , #188
